src/leiningen/i18n/bin/update-pot.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+set -eo pipefail
+
+POT_FILE='locales/messages.pot'
+COMPARE_POTS='dev-resources/i18n/bin/compare-POTs.sh'
+
+git_branch="$1";
+
+set -u
+
+if [[ -z "$git_branch" ]]; then
+  echo "FATAL: no git branch argument was supplied, so the updated POT can't be pushed" 1>&2
+  echo "usage: $0 remote_branch_name" 1>&2
+  exit 1
+fi
+
+if [[ ! -e "$POT_FILE" ]]; then
+  echo "FATAL: can't find '$POT_FILE'" 1>&2
```

Comment:
Isn't this script put into a fixed location ? You should be able to get an absolute path for the pot file from that, something along the lines of 
```
topdir=$(realpath $(dirname $0)/../../..)
pot_file="$topdir/locales/messages.pot"
```